### PR TITLE
Node.js Week2 Hometask

### PR DESCRIPTION
**Pull request by AnnaSolovykh**

### CODE REVIEW

**Review by leo-mutuku — APPROVED** [review]
writeWith Promises Await well implimented.
package.json correctly update as required

**Review by leo-mutuku — APPROVED** [review]
asynchrous write/read promises using then well implimented

File: 01-node-tutorial/answers/package.json
Comment (leo-mutuku) [inline 01-node-tutorial/answers/package.json:1]: well done nodemon & node start scripts

**Review by leo-mutuku — APPROVED** [review]

**Review by leo-mutuku — APPROVED** [review]
stream well implimented.

**Review by leo-mutuku — APPROVED** [review]
Event emiter well implimented

**Comment by leo-mutuku** [comment]
Great work! Thank you for keep is commit as descriptive as possible. Week2 tasks completed

**Comment by AnnaSolovykh** [comment]
> Great work! Thank you for keep is commit as descriptive as possible. Week2 tasks completed

Great! Thank you very much for the review!

**Pull request merged**